CONTRIBUTING.rst

```diff
@@ -0,0 +1,24 @@
+
+What can you do
+---------------
+
+If you like the project and think you could help with making it better, there are many ways you can do it:
+
+- Create a new issue for new feature proposal or a bug
+- Triage old issues that needs a refresh
+- Implement existing issues (there are quite some of them, choose whatever you like)
+- Help with improving the documentation (We still have work to do!)
+- Spread a word about the project to your colleagues, friends, blogs or any other channels
+- Any other things you could imagine
+
+Any contribution would be of great help and I will highly appreciate it! If you have any questions, please create a new issue. 
+
+Releasing a new package
+-----------------------
+
+Releasing a new version is done via Travis-CI.
```

Comment:
I think that this is no longer the case.... automatic release is in limbo.

It was removed here https://github.com/twisted/pydoctor/pull/235 and there was no followup PR or ticket.

I think that `Releasing a new package` can be removed and we will update the docs once we will implement / merge the auto-release PR.